packages/blocks/src/store/test/process-block-type.js

```diff
@@ -88,79 +88,7 @@ describe( 'processBlockType', () => {
 				radius: true,
 				style: true,
 				width: true,
-				__experimentalDefaultControls: {
```

Comment:
There was quite a bit of overlap and duplication between some of the test cases. I've consolidated some tests to keep things a little more concise as well as grouped relevant tests for deprecations within their own describe block.